src/vanilla/Model/CompositeTypeTS.cs

```diff
@@ -33,6 +33,35 @@ public override Property Add(Property item)
             return result;
         }
 
+        private List<CompositeTypeTS> polymorphicUnionCases;
+
+        /// <summary>Contains the immediate polymorphic children of this class.</summary>
+        public List<CompositeTypeTS> PolymorphicUnionCases
```

Comment:
Why is this a List? Are you expecting the caller of this property to add or remove values from this List?

Reply:
Same as above. Why isn't this an IEnumerable?